### PR TITLE
added .net 1.6.4 rns

### PR DESCRIPTION
**Pull request by jenniferopal**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [X] Maintenance/Chore

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -81,6 +81,12 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 ### .NET SDK
 
+#### Version 1.6.4
+
+ - The Evaluation logic refactored to use immutable principles, to aid in maintainacne and readability. (FFM-11057)
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:86]: Typo "maintainacne" :D

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -81,6 +81,12 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+  - If the target group cache is found to be in an invalid state, such that any groups that beloing to a flag are not in the cache, the SDK will asynchronously update the group cache and immediately return the default variation.
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:87]: Not sure if this bullet is aligned? looks indented by 1 space

File: release-notes/feature-flags.md (hunk 1 above)
Comment (jenniferopal) [reply release-notes/feature-flags.md:86]: Ah, good spot. I'll fix that.

**Review by erdirowlands — APPROVED** [review]
Looks good!

**Pull request merged**